out-adapter-persistence-jpa/src/main/java/gohigher/application/entity/ApplicationJpaEntity.java

```diff
@@ -97,6 +97,27 @@ public void addProcess(ApplicationProcessJpaEntity process) {
 		processes.add(process);
 	}
 
+	public void changeToDelete() {
+		this.deleted = true;
+	}
```

Comment:
```suggestion
	public void delete() {
		deleted = true;
	}
```

이 메서드는 테스트에서만 사용하는것 같은데, 나중에 사용될 가능성이 있어서 미리 만들고 테스트에서 활용하신 걸까요?

Reply:
delete를 위해 삭제된 엔티티를 만들기 위한 fixture 메서드를 추가할지, entity에 delete메서드를 만들지 고민중에 두번째 선택을 하였습니다.

현재는 테스트를 위한 메서드의 형태로 위치해있지만 테스트를 함에 있어 given절에서 해당 지원 공고가 삭제되었다는 것을 명시적으로 
표현할 수 있던 것 같습니다.

또한 이번 스프린트, 혹은 다음 스프린트에 지원서 삭제기능이 추가될 것이라 만들게 되었습니다..(현재만 생각했을 때는 좋은 방법은 아니라 생각합니다.)

그리고 Fixture를 위한 메서드를 하였을 때는 크게 가독성이 좋다고 느끼지 못하였으며 지원서 삭제 기능이 추가됨과 동시에 해당 fixture로직들은 불필요한 코드가 될 것인데 관리가 오히려 잘 안될것 같다는 생각이 들었습니다.

Reply:
지금 드는 생각은 JpaEntity의 delete 메서드를 사용한 `변경 감지` 방식으로 삭제를 할 것으로 예상하시는 것 같은데요!
변경감지를 사용하기 위해선 `JpaEntity를 조회해오는 것이 필수`입니다.
따라서, 직저 delete Query를 날리는 것이 좋아보인다는 생각이 드는데요!

삭제 API 개발 시에 다시 이야기 나누면 좋을 것 같습니다~!